src/pretix/plugins/sendmail/views.py

```diff
@@ -36,11 +36,18 @@ def form_valid(self, form):
         self.request.event.log_action('pretix.plugins.sendmail.sent', user=self.request.user, data=dict(
             form.cleaned_data))
 
+        failures = []
         for o in orders:
-            mail(o.email, form.cleaned_data['subject'], form.cleaned_data['message'],
-                 None, self.request.event, locale=o.locale, order=o)
-
-        messages.success(self.request, _('Your message will be sent to the selected users.'))
+            try:
+                mail(o.email, form.cleaned_data['subject'], form.cleaned_data['message'],
+                     None, self.request.event, locale=o.locale, order=o)
+            except Exception:
+                failures.append(o.email)
+
+        if failures:
+            messages.error(self.request, _('Failed to send mails to the following users: {}'.format(' '.join(failures))))
+        else:
+            messages.success(self.request, _('Your message has been sent to the selected users.'))
```

Comment:
No, in a production setup, it has only been queued for celery to send, therefore this change (and its translation) is wrong.